hardhat/scripts/start-ganache.sh

```diff
@@ -11,14 +11,17 @@ if [ -f "$DOTENV_FILE" ]; then
   export $(cat $DOTENV_FILE | sed 's/^#.*$//' | xargs)
```

Comment:
I had some unfilled in ENV_vars in my .env like
```
TELEGRAM_BOT_TOKEN=<your Telegram testing bot token>
```

and the `<` broke the script with error:
```
 ./hardhat/scripts/start-ganache.sh --deploy --exec ./scripts/ci.sh
./hardhat/scripts/start-ganache.sh: line 11: export: `token>': not a valid identifier
error Command failed with exit code 1.
```

So maybe we should change the default ENV_VAR for that one to not have `<` or anything that breaks this script when unused.

Reply:
Ah, good call